### ISSUE TITLE
In network.py line 45, an exception socket.gaierror is raised. 

### ISSUE THREAD

**Issue by jeffsilverm** [jeffsilverm/nbmdt#10]
In network.py line 45, an exception socket.gaierror is raised.  Two changes should be made:
1) When the exception is thrown, output the name of the offending host
2) Devise a sane alternative and implement it.

**Comment by jeffsilverm** [comment]
The first problem is that the default route is passed as the string 'default'.  So I added a test that detects 'default' and replaces it with '0.0.0.0'
However, there is a new problem and that is:
socket.gethostbyaddr raised a socket.gaierror exception on 10.0.3.0/24, continuing [Errno -2] Name or service not known
Tried to add the destination 10.0.3.0/24 to a route, and an OSError exception was raised
/24 is a netmask, and of course inet_aton is going to keel over and die when it sees it.  But it's 1:17 in the morning and I am going to pick this up later today.
